files/migrations.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+"""
+Runs migration scripts for this project
+
+Intended to be triggered by the %post rpm scriptlet rather than manually.
+"""
+
+import os
+import subprocess
+import sys
+from glob import glob
+
+VERSION_FILE = "/var/lib/{}/version"
```

Comment:
IMO all-uppercase globals should mostly be treated as constants, but these get redefined below. I would just define them in one place below.